Extension/src/LanguageServer/client.ts

```diff
@@ -962,35 +975,27 @@ export class DefaultClient implements Client {
         this.updateClientConfigurations();
     }
 
-    async promptSelectCompiler(command: boolean): Promise<void> {
+    async promptSelectCompiler(isCommand: boolean): Promise<void> {
         const selectCompiler: string = localize("selectCompiler.string", "Select Compiler");
         const confirmCompiler: string = localize("confirmCompiler.string", "Yes");
         const settings: OtherSettings = new OtherSettings();
         if (compilerDefaults.compilerPath !== "") {
-            if (!command && (compilerDefaults.compilerPath !== undefined)) {
+            if (!isCommand && (compilerDefaults.compilerPath !== undefined)) {
                 const value: string | undefined = await vscode.window.showInformationMessage(localize("selectCompiler.message", "The compiler {0} was found on this computer. Do you want to configure it for IntelliSense?", compilerDefaults.compilerPath), confirmCompiler, selectCompiler);
```

Comment:
Can you remove "on this computer" -- it's extra text the user shouldn't need to read.

Reply:
@tarik-brown What was the resolution for this?

Reply:
check latest changes